src/_guides/language/language-tour.md

```diff
@@ -2345,9 +2345,21 @@ const pointAndLine = {
 };
 {% endprettify %}
 
+If a constant constructor is outside of a constant context
```

Comment:
Do you define "constant context" anywhere? If not, it would be good to show an example where you don't need `const` but it *does* create a const object, like:

```dart
const foo = Foo(); // "const Foo()" not needed here.
```

Reply:
We don't explicitly define it, but we show it. E.g. in the example before, line 2341 says "only one const, which establishes the constant context".